consensus/babe/babe.go

```diff
@@ -50,6 +52,8 @@ type SessionConfig struct {
 	NewBlocks chan<- types.Block
 }
 
+const MaxBlockSize uint = 4*1024*1024 + 512
```

Comment:
maybe a comment here to explain `4*1024*1024 + 512`

Reply:
@dziabko where did this come from?

Reply:
gonna remove this for now since I think the algo is different.